README.md

```diff
@@ -29,7 +29,7 @@ For example, if you made are making an etcd cluster to be used by a service call
 
     senza create https://raw.github.com/zalando/stups-etcd-cluster/master/etcd-cluster.yaml releaseetcd \
                                    HostedZone=elephant.example.org \
-                                   DockerImage=registry.opensource.zalan.do/acid/etcd-cluster:3.0.8-p12
+                                   DockerImage=registry.opensource.zalan.do/acid/etcd-cluster:3.0.10-p12
```

Comment:
Do we not reset the patch counter after minor version release?  I mean will the package built after this is merged really have the `-p12` part, instead of `-p1`?